code/modules/materials/Mat_Materials.dm

```diff
@@ -669,6 +669,7 @@ ABSTRACT_TYPE(/datum/material/metal)
 		setProperty("electrical", 6)
 		setProperty("radioactive", 5)
 		setProperty("hard", 2)
+		addTrigger(TRIGGERS_ON_TEMP, new /datum/materialProc/radioactive_temp())
```

Comment:
Maybe consider adding this hackily to `setProperty` so any future radioactive materials get it automatically?

Reply:
`onAdded()`